content/pt-br/docs/tasks/access-application-cluster/configure-dns-cluster.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Configurar DNS para um Cluster
+weight: 120
+content_type: concept
+---
+
+<!-- overview -->
+Kubernetes oferece um complemento de DNS para os clusters, que a maioria dos ambientes suportados o tem habilitado por padrão. Na versão do Kubernetes 1.11 e posterior, o CoreDNS é recomendado e instalado por padrão com o kubeadm.
```

Comment:
Some suggestions:
* "o tem habilitado" is sounding a bit weird. How would you feel about using "habilita por padrão"?
* I think it would be good to add the article "o" before Kubernetes in this first sentence ("O Kubernetes...").

Reply:
Hello :smile:, Sorry for the delay.

For the first suggestion, could be something like "habilitam por padrão" ?

Or is it still kinda weird ?

Reply:
Yes, "habilitam por padrão" sounds great! 💯